Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Registration.hs

```diff
@@ -443,6 +448,8 @@ createPerson req mobileNumber notificationToken mbBundleVersion mbClientVersion
           flowStatus = DPFS.IDLE,
           updatedAt = person.updatedAt
         }
+    makePersonStats :: (EncFlow m r, EsqDBFlow m r, DB.EsqDBReplicaFlow m r, CacheFlow m r, CoreMetrics m) => SP.Person -> Id DMerchant.Merchant -> m DPS.PersonStats
+    makePersonStats person mId = SLP.backfillPersonStats person.id mId
```

Comment:
Why do we need to backfill data for the newly created person? There will be no data for him, right?

Reply:
modified .
just personstats needs to be created for new driver